template/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+graft src
```

Comment:
I think this file should be present only if `setuptools` is selected?

Reply:
Yes, this file isn't needed for `hatch`, @santacodes. It should be present for just setuptools-without-scm. For setuptools-with-scm, you may refer to my earlier comment: https://github.com/pybamm-team/pybamm-cookiecutter/pull/37#pullrequestreview-2217516990